cea/datamanagement/format_helper/cea4_migrate.py

```
@@ -0,0 +1,411 @@
+"""
+Mirgate the format of the input data to CEA-4 format after verification.
+
+"""
+
+import cea.inputlocator
+import os
+import cea.config
+import time
+import pandas as pd
+import geopandas as gpd
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)